src/firmware_new/boards/frankie_v1/lwip.c

```diff
@@ -27,6 +27,8 @@
 #endif /* MDK ARM Compiler */
 
 /* USER CODE BEGIN 0 */
+#include "constants.h"
+#include "udp_multicast.h"
```

Comment:
We should not use relative includes for files we control. Should be `firmware_new/boards/frankie_v1/constants.h`

Reply:
fixed